client/Components/Pages/Signup.jsx

```diff
@@ -39,30 +42,43 @@ class Signup extends Component {
     });
   }
 
-  handleSubmit(e) {
-    e.preventDefault();
+  handleSubmit(event) {
```

Comment:
I see you are making the event argument name clear. Good one.

Reply:
Thanks